wurst/systems/core/Tribe.wurst

```diff
@@ -5,6 +5,30 @@ import TribeUtils
 import Lodash
 import PlayerExtensions
 import LegacyColors
+import GameConfig
+import GameMode
+
+let listeners = CreateTrigger()
+
+public function registerTribeInitializationFinishEvent(code listener)
+    listeners.addAction(listener)
+
+let names = asList(
```

Comment:
https://wowwiki.fandom.com/wiki/Troll_tribes

Reply:
I saw that. I didn't want to go full WoW. Besides, I like mine more.